.aws/src/backfillLambda.ts

```diff
@@ -79,10 +80,62 @@ export class BackfillLambda extends Resource {
           },
         ],
         alarms: {
-          // TODO: set better alarm values
+          errors: {
+            // The backfill lambda will likely only run for 10-15 minutes total,
+            // so having more than one evaluation period is not necessary.
+            evaluationPeriods: 1,
+            comparisonOperator: 'GreaterThanOrEqualToThreshold',
+            period: 300, // 5 minutes
+            threshold: 50,
+            // TODO: remove this alarm from Dev when testing is complete
+            // actions: config.isDev
+            //   ? []
+            //   : [pagerDuty!.snsNonCriticalAlarmTopic.arn],
+
+            // For now, we need alerts in Dev, too
+            actions: [pagerDuty.snsNonCriticalAlarmTopic.arn],
+          },
+          invocations: {
+            evaluationPeriods: 1,
+            comparisonOperator: 'LessThanLowerOrGreaterThanUpperThreshold',
+            period: 300, // 5 minutes
+            // approx. 10 batches of 20 records each, it in an estimated ~1000 batches within 5 minutes.
+            threshold: 10,
+            // TODO: disable this alarm for Dev when testing is complete
+            actions: [pagerDuty.snsNonCriticalAlarmTopic.arn],
+          },
         },
       },
       tags: config.tags,
     });
+
+    this.createDLQAlarm();
+  }
+
+  /**
+   * Create an alarm for the Dead Letter Queue. This is the only place this is going
+   * to be used so far - if it's ever reused, it should be moved to Terraform Modules.
+   */
+  createDLQAlarm() {
+    return new cloudwatch.CloudwatchMetricAlarm(this, 'backfill-dlq-alarm', {
+      alarmName: config.prefix + '-Backfill-DLQ-Alarm',
+      alarmDescription:
+        'Alert on more than 5% of backfilled records ending up in the DLQ.',
+      namespace: 'AWS/SQS',
+      metricName: 'ApproximateNumberOfMessagesVisible',
+      dimensions: {
+        QueueName: config.prefix + '-Backfill-Lambda-Queue-Deadletter',
+      },
+      comparisonOperator: 'GreaterThanOrEqualToThreshold',
+      evaluationPeriods: 1,
+      // 5 minutes
+      period: 300,
```

Comment:
as we decided to throttle lambda upto 10 concurrent instance max, should this time be more than 300s?